docs/docsgen/source/intro/python.rst

```diff
@@ -1179,6 +1180,279 @@ a Reshape operator. Shape inference only works if the shape is constant.
 If not constant, the shape cannot be easily inferred unless
 the following nodes expect specific shape.
 
+Evaluation and Runtime
+======================
+
+An ONNX model can be used to move a model from one library to another
+or it can be directly evaluated with an ONNX runtime.
+:epkg:`onnxruntime` is one efficient option. It is available in many platforms.
+It is optimized for fast inference. Its coverage can be tracked on
+`ONNX Backend Dashboard <http://onnx.ai/backend-scoreboard/>`_.
+*onnx* implements a python runtime
+useful to help understand a model.
```

Comment:
Add `but is not intended for production use`.

Reply:
done